### PR TITLE
Aliased subworkflows test

### PR DESCRIPTION
**Pull request by cjllanwarne**

### CODE REVIEW

File: src/main/resources/standardTestCases/aliased_subworkflows.test [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+files {
+  wdl: aliased_subworkflows/aliased_subworkflows.wdl
+  imports: [
+    aliased_subworkflows/subworkflow.wdl
+  ]
+}
+
+metadata {
+  workflowName: aliased_subworkflows
```
Comment (ruchim) [inline src/main/resources/standardTestCases/aliased_subworkflows.test:13]: TOL: maybe worth adding the outputs for Array[int] results

**Comment by ruchim** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/123/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

Now that I understand the motivation for this PR is https://github.com/broadinstitute/cromwell/issues/1785, maybe its worth adding in the test case you expect to pass but is broken, and setting it to ignore and we can re-enable it when the issue is addressed?

File: src/main/resources/standardTestCases/aliased_subworkflows.test (hunk 1 above)
Comment (cjllanwarne) [reply src/main/resources/standardTestCases/aliased_subworkflows.test:13]: Done!

**Comment by mcovarr** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/123/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

**Pull request merged**